sources/packages/web/src/components/common/Root.vue

```diff
@@ -0,0 +1,115 @@
+<template>
```

Comment:
Can we name the component as something else related to the logoff timeout control?
I got the idea that this will be at the root of the application, but we can build other ones that will also be on the root with different purposes.

Reply:
mm, ya sure `IdleTimeChecker`. or do you guys  have any better name suggestion

Reply:
I like it 😉